src/test/java/org/sonar/scm/git/blame/FileBlamerTest.java

```diff
@@ -56,13 +54,18 @@ public class FileBlamerTest {
   @Before
   public void before() {
     when(revCommit.getName()).thenReturn(ANY_COMMIT_NAME);
+
     PersonIdent personIdent = mock(PersonIdent.class);
     when(revCommit.getAuthorIdent()).thenReturn(personIdent);
     when(personIdent.getEmailAddress()).thenReturn(ANY_EMAIL);
     when(personIdent.getWhen()).thenReturn(ANY_DATE);
+
+    PersonIdent personIdent2 = mock(PersonIdent.class);
+    when(revCommit.getCommitterIdent()).thenReturn(personIdent2);
+    when(personIdent2.getEmailAddress()).thenReturn("another@email.com");
+    when(personIdent2.getWhen()).thenReturn(ANY_DATE);
```

Comment:
Dates should be different since we take the committer date instead of author date

Reply:
done